content/response-to-reviewers.md

```diff
@@ -0,0 +1,134 @@
+
+> - Table 1: What is the organizing principle behind the order of the databases? Would it be better to order them alphabetically?
+> - "or though the use"?
+> - "into a subphrases"?
+> - Table 3: Maybe define or spell out the Types of Sentences.
+> - "This values within"?
+> - "uses a matrices"?
+> - "other effots"?
+> - "then determines how"?
+> - Figure 5b: Font too small
+> - "goal of recommend safe drugs"
+
+**We would like to thank the reviewer for noticing these typos and we have fixed them in this new revision.**
+
+## Reviewer 2
+
+>The manuscript attempts to review methods for knowledge graphs and their biomedical applications. 
+>The review is too general. 
+>It tries to combine disparate lines of research ranging from dimensionality reduction, manifold learning, to graph community detection and frame it under knowledge graph learning. 
+>Although no established definition of knowledge graph exists, an essential property of a knowledge graph is that it models relations between nodes, which don't apply to dimensionality methods (SVD,PCA,,etc.) and other methods such as node2vec. 
+>Almost all of the discussions in this manuscript deals with methods that don't model relations. 
+>The only part that correctly refers to a class knowledge graph methods is translational distance methods.  
+
+**We appreciate the reviewer's perspective; however, we emphasize that the focus of this review is to discuss the pros and cons of building knowledge graphs and representing them in a low dimensional space.
+This space is intended to capture the essence of a knowledge graph and is dependent on the node and edge structure of a graph.
+Once knowledge graphs are represented in this space, they can no longer be considered "true" knowledge graphs. 
+The methods discussed in our review are focused on how this low dimensional space is constructed and we updated our text to be more explicit on how knowledge graphs are used as input.
+**
+
+>The following are some comments: 
+
+>What specifically do you mean by techniques that represent KGs and machine learning methods that are used to learn low-dimensional vectors?
+
+**When we say, "techniques that represent knowledge graphs in a low dimensional space", we allude to methods such as node2vec and translational distance models that associate nodes and possibly edges to dense vectors.
+These vectors implicitly capture the essence of a knowledge graph itself and machine learning methods can utilize these vectors to perform tasks such as link prediction for biomedical applications.**
+
+>at least references 1, 2, 5,6 and 2 are concerned with methods for graphs not necessarily knowledge graphs, in which the edge label (i.e., relation is essential to its definition).
+
+**We define biomedical knowledge graphs as the following: "a resource that integrates one or more expert-derived sources of information into a graph where nodes represent biomedical entities and edges represent relationships between two entities.".

```

Comment:
```suggestion
**As there was no single, well-accepted definition of a knowledge graph we had to select one from the literature.
We selected a relatively permissive definition because we felt that many recent methodological advances had been demonstrated primarily on more general graphs of biomedical entities extracted from biomedical knowledge bases.
We specifically defined a biomedical knowledge graphs as: "a resource that integrates one or more expert-derived sources of information into a graph where nodes represent biomedical entities and edges represent relationships between two entities."
We agree that not all of these references were about biomedical knowledge graphs.
We have now clarified the first paragraph to discuss _graphs_, before diving into our definition of biomedical knowledge graphs in the second paragraph.**
```